cli/core/pkg/command/root.go

```diff
@@ -47,7 +48,7 @@ func NewRootCmd() (*cobra.Command, error) {
 	if ni != "" || strings.EqualFold(forceNoInit, "true") {
 		noInit = true
 	}
-	if os.Getenv("TANZU_CLI_NO_COLOR") != "" {
+	if component.IsTTYEnabled() {
 		color = false
 	}
```

Comment:
How about we remove this code and adjust the below to use the new cli.Bold(..) you just adapted?
We'd have to make those functions public though.